datapreview.tex

```diff
@@ -61,91 +64,38 @@ \subsection{Data Preview 1}
-        \caption{Rubin SV 38 7}
-        \label{fig:img2}
-    \end{subfigure}
-    \hfill
-    \begin{subfigure}[b]{0.3\textwidth}
-        \centering
-        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_fornax_dsph_r}
-        \caption{Fornax dSph}
-        \label{fig:img3}
-    \end{subfigure}
-    \vspace{1em}  % Vertical space between rows
-     \begin{subfigure}[b]{0.3\textwidth}
-        \centering
-        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_ecdfs_r}
-        \caption{ECDFS}
-        \label{fig:img4}
-    \end{subfigure}
-    \hfill
-    \vspace{1em}  
-    \begin{subfigure}[b]{0.3\textwidth}
-        \centering
-        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_edfs_r}
-        \caption{EDFS}
-        \label{fig:img6}
-    \end{subfigure}
-    \hfill
-    \begin{subfigure}[b]{0.3\textwidth}
-        \centering
-        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_rubin_sv_95_-25_r}
-        \caption{Rubin SV 95 -25}
-        \label{fig:img7}
-    \end{subfigure}
-    \hfill
-    \begin{subfigure}[b]{0.3\textwidth}
-        \centering
-        \includegraphics[width=\textwidth]{figures/comcam_psf_maglim_seagull_r}
-        \caption{Seagull}
-        \label{fig:img8}
-    \end{subfigure}
-     \caption{Cumulative imaging depth expressed in terms of the $S/N=5$ limiting magnitude for unresolved sources for seven ComCam Deep Drilling Fields.}
-     \label{fig:dp1_fields_psf_maglim}
-\end{figure}
-
-The  processing and preparation of ComCam data for DP1 will take place during the first half of 2025, with an expected DP1 release data in June or July 2025. 
+The  processing and preparation of LSSTComCam data for DP1 took place during the first half of 2025. 
+The planned release date for DP1 is 30 June 2025. 
 The planned data products for DP1 are presented in Table~\ref{tab:data-preview-summary}.
-DP1 is expected to include of the order of 2000 science images.
-The processing of crowded fields in the centers of 47 Tuc and Fornax is on a best-effort basis for DP1.
-The center of 47 Tuc is saturated and so no coadd images will be produced for this region.
+DP1 is expected to include of the order of 2000 exposures.
+The processing of crowded fields in the centers of 47 Tuc and Fornax is on a best-effort basis.
+The center of 47 Tuc is saturated and so no coadd images are expected to be produced for this region.
+Solar System catalogs will contain observations of known objects from the MPC Orbit Catalog (MPCORB) tables as well as any new discoveries. 
+The snapshot of the MPCORB Catalog tables included in DP1 is taken from around the time of DRP processing for DP1. 
+
+As the processing progresses, it is conceivable that some individual exposures might not meet the quality control standard of being ``science-grade, astronomically useful'' images, 
+and consequently, the exact number of exposures and coadd depths in DP1 may change.
```

Comment:
```suggestion
and consequently, the exact number of exposures and coadd depths in DP1 may change before the actual release.
```

(We're not trying to claim it will change after the release, I hope?)